.github/dependabot.yml

```diff
@@ -173,3 +173,20 @@ updates:
       csm-installer:
         patterns:
           - "*"
+
+  # installation-wizard packages
```

Comment:
Will also need a separate PR to add these changes to the main branch since that is where the dependabot action will run. Example: https://github.com/dell/helm-charts/pull/491/files

Reply:
Yup, you are too quick!

PR for main on dependabot action: https://github.com/dell/helm-charts/pull/538.